src/qt/multisenddialog.cpp

```diff
@@ -71,6 +79,14 @@ void MultiSendDialog::on_viewButton_clicked()
 
     for (int i = 0; i < (int)pwalletMain->vMultiSend.size(); i++) {
         pMultiSend = pwalletMain->vMultiSend[i];
+        if (model && model->getAddressTableModel()) {
+            std::string associatedLabel;
+            associatedLabel = model->getAddressTableModel()->labelForAddress(pMultiSend.first.c_str()).toStdString();
+            if (associatedLabel != "(no label)") {
```

Comment:
Is this gonna fly after translation? Maybe use a bool to know if it is set instead of comparing a string.

Reply:
That is a good point. I am changing it print the label even if there is no label assigned. This way the users will see if correct label in the correct language.